pocs/tests/test_pocs.py

```diff
@@ -192,13 +191,32 @@ def test_is_weather_safe_no_simulator(pocs):
     assert pocs.is_weather_safe() is False
 
 
+def wait_for_message(sub, type=None, attr=None, value=None):
```

Comment:
I think I'm guilty of this, but `type` is a built-in function and probably shouldn't be used as a variable name. (See also comments about `id` below).